src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/layout/fetch/fetchFormLayoutSagas.ts

```diff
@@ -8,11 +8,12 @@ import QueueActions from '../../../../shared/resources/queue/queueActions';
 
 function* fetchFormLayoutSaga({ url }: IFetchFormLayout): SagaIterator {
   try {
-    const { data }: any = yield call(get, url);
+    const { data, autoSave }: any = yield call(get, url);
```

Comment:
We already have `data.layout`, why not use `data.autoSave` instead of having this as a separate object?

Reply:
I just felt it was weird that we wrapped our formlayout in another object? So formlayout looks like this `{ autoSave: true, data: { layout: []}`. I'm more unsure of why we wrap the layout in a data object? 

No strong preferences, maybe it should live in a config object.

Reply:
But I don't feel its natural that it goes under data

Reply:
Fixed.